### ISSUE TITLE
tfjs-models/coco-ssd: only objects in left half of an image are detected

### ISSUE THREAD

**Issue by KjelleJ** [tensorflow/tfjs#2391]
#### TensorFlow.js version
    "@tensorflow-models/coco-ssd": "^2.0.0",
    "@tensorflow/tfjs-converter": "^1.2.11",
    "@tensorflow/tfjs-core": "^1.2.11",

#### Browser version
Google Chrome Version 78.0.3904.97

#### Describe the problem or feature request
It looks like coco-ssd only detects objects in the left half (approx) of an image. Is it really a bug - it is too obvious...? Or is it a feature - I don't know coco-ssd.

#### Code to reproduce the bug / link to feature request
You can see this when running the demo app. Replace image2 (the dogs) with image1 flipped horizontally (left will be right etc). 4 objects are detected for image1 and NO objects for image2 (the flipped image1).

**Comment by rthadur** [comment]
@KjelleJ thanks for the report, can you please provide more details, 
1) tfjs version 
2) code to reproduce 
3) use case if applicable 
Thank you 
CC @pyu10055 @dsmilkov

**Comment by KjelleJ** [comment]
1. 0.15.4
2. https://github.com/tensorflow/tfjs-models/tree/master/coco-ssd/demo
3. You can see this when running the demo app. Replace image2 (the dogs) with image1 flipped 
 so left will be right etc. 4 objects are detected for image1 and NO objects for image2 (the flipped image1)

**Comment by KjelleJ** [comment]
Some screenshots from my app to show the problems with the right part of an image.

![katt2](https://user-images.githubusercontent.com/48243574/68934487-0bf0ad80-0797-11ea-9812-4dd8c34718db.png)
![katt1](https://user-images.githubusercontent.com/48243574/68934508-1448e880-0797-11ea-9251-af2ec5b98349.png)
![cellphone](https://user-images.githubusercontent.com/48243574/68934549-288ce580-0797-11ea-80d4-b99db81cd3c8.png)

**Comment by rthadur** [comment]
@KjelleJ thank you, is there a reason you are using older version ? did you try this with latest tfjs version ?

**Comment by KjelleJ** [comment]
No there is no reason. I tried to update the demo app but the old tfjs version is still used. But I have the same problem with my app using a newer tfjs version (1.2.11 or so). 
![park1](https://user-images.githubusercontent.com/48243574/68962646-55f88400-07d5-11ea-8a5f-2d9af39489bc.png)
![park2](https://user-images.githubusercontent.com/48243574/68962652-5abd3800-07d5-11ea-94af-679f8580f287.png)

**Comment by pyu10055** [comment]
I have tried with the latest versions:
  "dependencies": {
     "@tensorflow-models/coco-ssd": "2.0.0",
     "@tensorflow/tfjs-core": "1.3.2",
     "@tensorflow/tfjs-converter": "1.3.2",
     "stats.js": "^0.17.0"
   },
The flipped image detection looks correct
![screenshot](https://user-images.githubusercontent.com/4018/68976744-5aa84280-07ab-11ea-9cb6-329bf86efd12.png)

**Comment by KjelleJ** [comment]
@pyu10055 I tried your  proposal (latest versions) but it still does not work. I downloaded tfjs-models in March and just tried the demo app the other day following the instructions in Readme for the demo app . I tried the demo app after having the same problem with my app.

Now I downloaded tfjs-models once more and the demo app works! Thank you! I still have to examine why my app does not work. I used the demo app as a starter for my app...
